components/sidebar_right/sidebar_right.jsx

```diff
@@ -93,7 +96,7 @@ export default class SidebarRight extends React.PureComponent {
                 } else {
                     this.props.actions.openAtPrevious(this.previous);
                 }
-            } else if (e.altKey && Utils.isKeyPressed(e, Constants.KeyCodes.I)) {
+            } else if (Utils.isKeyPressed(e, Constants.KeyCodes.I) && (channelInfoShortcutMac || channelInfoShortcut)) {
```

Comment:
nit: maybe a ternary here could make it more direct and executed only when needed.